src/main/java/in/projecteka/consentmanager/user/OtpServiceClient.java

```diff
@@ -0,0 +1,55 @@
+package in.projecteka.consentmanager.user;
+
+import in.projecteka.consentmanager.clients.ClientError;
+import in.projecteka.consentmanager.user.model.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+
+public class OtpServiceClient {
+
+    private final WebClient.Builder webClientBuilder;
+    private final OtpServiceProperties otpServiceProperties;
+    private AuthenticatorService authenticatorService;
+
+    public OtpServiceClient(WebClient.Builder webClientBuilder,
+                            OtpServiceProperties otpServiceProperties,
+                            AuthenticatorService authenticatorService) {
+        this.webClientBuilder = webClientBuilder;
+        this.webClientBuilder.baseUrl(otpServiceProperties.getUrl());
+        this.otpServiceProperties = otpServiceProperties;
+        this.authenticatorService = authenticatorService;
+    }
+
+    public Mono<TemporarySession> sendOtpTo(OtpRequest requestBody) {
```

Comment:
```suggestion
    public Mono<TemporarySession> sendOtpTo(OtpRequest requestBody) {
```
```suggestion
    public Mono<TemporarySession> send(OtpRequest requestBody) {
```
No need of OtpTo as there is no mobileNumber parameter explicitly mentioned.